.github/workflows/actionlint.yml

```diff
@@ -23,8 +23,12 @@ jobs:
   actionlint:
     runs-on: ubuntu-latest
     steps:
+      - name: "Harden Runner"
```

Comment:
There is a much better solution for actionlint. See https://github.com/instructlab/community/pull/302 for example.